lib/fine-types/app/Options/Common.hs

```diff
@@ -0,0 +1,45 @@
+{-# LANGUAGE ApplicativeDo #-}
+{-# LANGUAGE DuplicateRecordFields #-}
```

Comment:
```suggestion
```

This extension appears to be unused here. 🤔 (I'm generally ok with this extension.)